components/inflector.rst

```diff
@@ -0,0 +1,53 @@
+.. index::
+   single: Inflector
+   single: Components; Inflector
+
+The Inflector Component
+=======================
+
+    The Inflector component converts English words between singular and plural forms.
```

Comment:
In french, when using adjectives or the name of the language, we use lower case, upper case is for inhabitant. Is the rule different in "english" ? :)